### ISSUE TITLE
Convert RefreshButton to fontawesome-5

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#685]
From https://github.com/phetsims/sherpa/issues/83 ...

This is going to be a bit of a project.  RefreshButton has option `iconScale`, which is used by many clients.  And the scale is very different between fontawesome-4 and fontawesome-5.  

So that we don't have to deal with this again if we ever change icons sources, I'm going to replace `iconScale` with `iconHeight`, and make RefreshButton responsible for computing the proper scale. 

The fontawesome-5 file to use is syncAltSolidShape.js.

**Comment by pixelzoom** [comment]
Clients of RefreshButton, and the current height of their RefreshButtonIcon:

build-a-molecule: 19.200000000000003
density-buoyancy-common: 19.200000000000003
equality-explorer: 23.04
expression-exchange: 26.88
fractions-common: 26.88
function-builder: 38.400000000000006
function-build-basics: 38.400000000000006
scenery-phet: 38.400000000000006
unit-rates: 13.824

**Comment by pixelzoom** [comment]
API and client call sites were changed in the above commits. Instead of weird values like 38.400000000000006, I rounded to 38.  Here are the values that I used (and visually inspected) for each client:

build-a-molecule: 20
density-buoyancy-common: 20
equality-explorer: 23
expression-exchange: 27
fractions-common: 27
function-builder: 38
function-build-basics: 38
scenery-phet: 35
unit-rates: 14

Closing.
